### PR TITLE
Add documentation sql database

### PR DESCRIPTION
**Pull request by JanMaartenvanDoorn**

### CODE REVIEW

File: docs/sql_database_diagram.rst [hunk 1]
```diff
@@ -2,10 +2,447 @@
     SPDX-FileCopyrightText: 2017-2023 Contributors to the OpenSTEF project <korte.termijn.prognoses@alliander.com>
     SPDX-License-Identifier: MPL-2.0
 
-Overview of relational database schema
+Overview of relational database
 ============================================================
 
 OpenSTEF uses a relational database to store information about prediction jobs and measurements. An ER diagram of this database is shown bellow.
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:8]: below

File: docs/sql_database_diagram.rst [hunk 2]
```diff
@@ -2,10 +2,447 @@
...
+| id             | int            | API key id      | 199               |
++----------------+----------------+-----------------+-------------------+
+| name           | chr            |                 | Measurements      |
++----------------+----------------+-----------------+-------------------+
+| apiKey         | chr            | API key value   | uuid-Measurements |
++----------------+----------------+-----------------+-------------------+
+
+todolist
+---------
+This table is empty in openstef-reference.
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:378]: Maybe add it?

File: docs/sql_database_diagram.rst [hunk 3]
```diff
@@ -2,10 +2,447 @@
...
+| function             | chr         | functions to   |                |
+|                      |             | execute        |                |
++----------------------+-------------+----------------+----------------+
+| args                 | chr         | arguments of   |                |
+|                      |             | the functions  |                |
++----------------------+-------------+----------------+----------------+
+| inprogress           | int         |                |                |
++----------------------+-------------+----------------+----------------+
+
+Manually add a task besides those scheduled by Kubernetes. The list is
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:396]: Elaborate some more on this?

File: docs/sql_database_diagram.rst [hunk 4]
```diff
@@ -2,10 +2,447 @@
...
+automatically checked by Kubernetes.
+
+weatherforecastlocations
+------------------------
+Contains the locations of the weather stations. Not used in OpenSTEF.
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:401]: Is it actually used in Openstef?

File: docs/sql_database_diagram.rst [hunk 5]
```diff
@@ -2,10 +2,447 @@
...
+| lon            | double         |                 | 5.8             |
++----------------+----------------+-----------------+-----------------+
+| country        | chr            |                 | NL              |
++----------------+----------------+-----------------+-----------------+
+| active         | int            |                 | 1               |
++----------------+----------------+-----------------+-----------------+
+
+windspecs
+---------
+This table is empty in openstef-reference. Contains the information for
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:422]: Maybe add some data to this table in openstef-reference?

File: docs/sql_database_diagram.rst [hunk 6]
```diff
@@ -2,10 +2,447 @@
...
+| n_turbines   | int          | number of wind turbines |             |
++--------------+--------------+-------------------------+-------------+
+| hub_height   | int          | height of the turbines  |             |
+|              |              | (m)                     |             |
++--------------+--------------+-------------------------+-------------+
+
+The hub height is used to extrapolate the wind speed forecast at the
+correct height.
+
+Cf calculate_windspeed_at_hubheight in
```
Comment (touronaug) [inline docs/sql_database_diagram.rst:447]: Keep this?

**Review by touronaug — CHANGES_REQUESTED** [review]

File: docs/sql_database_diagram.rst [hunk 7]
```diff
@@ -2,10 +2,447 @@
...
+|               |            | corresponding to 50% of |                |
+|               |            | rated power (m/s)       |                |
++---------------+------------+-------------------------+----------------+
+| steepness     | float      | See formula             | 0.76           |
++---------------+------------+-------------------------+----------------+
+
+In openstef/feature_engineering/weather_features.py, the power delivered
+by a wind turbine is computed as
+
+.. math:: P(v) = \frac{P_{rated}}{1 + e^{- k(v - c)}},
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:124]: Theses rst files will be converted to HTML files that do support formulas.On my local machine these formulas are correctly displayed in the HTML output.

File: docs/sql_database_diagram.rst [hunk 8]
```diff
@@ -2,10 +2,447 @@
...
++--------------------+----------+------------------------+-----------------+
+| forecast_type      | chr      | type of forecast       | demand          |
++--------------------+----------+------------------------+-----------------+
+| model              | chr      | type of model          | xgb             |
++--------------------+----------+------------------------+-----------------+
+| created            | datetime | creation datetime of   | 2019-05-16      |
+|                    |          | the prediction job     | 14:53:38        |
++--------------------+----------+------------------------+-----------------+
+| active             | int      | 0 = off; 1 = on; 2 =   | 1               |
+|                    |          | automatic              |                 |
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:202]: This seems to not be used anymore, I deleted it.

File: docs/sql_database_diagram.rst [hunk 9]
```diff
@@ -2,10 +2,447 @@
...
++--------------------+----------+------------------------+-----------------+
+| horizon_minutes    | int      | max forecast horizon   | 2880            |
+|                    |          | (minutes)              |                 |
++--------------------+----------+------------------------+-----------------+
+| resolution_minutes | int      | time resolution of     | 15              |
+|                    |          | forecasts (minutes)    |                 |
++--------------------+----------+------------------------+-----------------+
+| train_components   | bool     | use splitting          | 1               |
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:210]: Yes it is still used. I eleborated documentation.

File: docs/sql_database_diagram.rst [hunk 10]
```diff
@@ -2,10 +2,447 @@
...
++-----------------+----------+--------------------------+--------------+
+| value           | char     | hyper-parameter value    | 0.91         |
++-----------------+----------+--------------------------+--------------+
+| created         | datetime | datetime when the value  | 2021-04-30   |
+|                 |          | of the hyper-parameter   | 13:04:00     |
+|                 |          | has been added to the    |              |
+|                 |          | table                    |              |
++-----------------+----------+--------------------------+--------------+
+
+Deprecated: Hyper-parameters are managed by MLflow.
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:169]: No, this is an important table to link the relevant systems to one or multiple prediction jobs

File: docs/sql_database_diagram.rst [hunk 11]
```diff
@@ -2,10 +2,447 @@
...
++------------+----------+------------------------------------+-------------+
+| **Name**   | **Type** | **Comment**                        | **Example** |
++============+==========+====================================+=============+
+| pid        | int      | prediction job id                  |             |
++------------+----------+------------------------------------+-------------+
+| lat        | double   | latitude                           |             |
++------------+----------+------------------------------------+-------------+
+| lon        | double   | longitude                          |             |
++------------+----------+------------------------------------+-------------+
+| radius     | int      | radius in km                       |             |
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:287]: Not sure how this is used exactly as this part is pretty old. @FrankKr probably knows this.

File: docs/sql_database_diagram.rst (hunk 4 above)
Comment (JanMaartenvanDoorn) [reply docs/sql_database_diagram.rst:401]: Yes, it is used for storing and retrieving weather data

**Review by touronaug — APPROVED** [review]

**Pull request merged**